cmd/otelcol/Dockerfile.windows

```
@@ -27,6 +27,26 @@ RUN Add-Type -AssemblyName System.IO.Compression.FileSystem ;\
     [System.IO.Compression.ZipFile]::ExtractToDirectory('agent-bundle_windows_amd64.zip', '.')
 RUN Remove-Item agent-bundle_windows_amd64.zip -force
 
+# Download JMX Metric Gatherer
+ARG JMX_METRIC_GATHERER_RELEASE
+RUN New-Item -Path "C:\\" -Name "opt" -ItemType "directory"
+RUN Invoke-WebRequest -Uri "https://github.com/open-telemetry/opentelemetry-java-contrib/releases/download/v${env:JMX_METRIC_GATHERER_RELEASE}/opentelemetry-jmx-metrics.jar" -Outfile "c:\opt\opentelemetry-java-contrib-jmx-metrics.jar"
+
+ARG JDK_VERSION="11.0.20_8"
+ARG OPENJDK_BASE_URL="https://github.com/adoptium/temurin11-binaries/releases/download"
+ENV JAVA_HOME="C:\java\jdk-$JDK_VERSION"
+RUN [Environment]::SetEnvironmentVariable('JAVA_HOME', "${Env:JAVA_HOME}.replace('_', '+')")
```

It may be simplest/safest to just use a set path, something like:
```suggestion
ENV JAVA_HOME="C:\java\jdk"
```
And then after the zip is downloaded and extracted, just rename the extracted directory:
```
RUN Move-Item C:\java\jdk-* C:\java\jdk
```

---

Sounds good to me. Looks like `Move-Item` [does](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.management/move-item?view=powershell-7.3) support regex, so I'll try this.